test/conftest.py

```diff
@@ -360,83 +379,85 @@ def inner(revs, bugs, message=None, bookmarks="mozilla/autoland"):
         if message is None:
             message = b"\n".join(msg)
         return hg_commit(hg_gecko_upstream, message, bookmarks)
+
     return inner
 
 
 @pytest.fixture
 def gecko_worktree(env, git_gecko):
-    path = os.path.join(env.config["root"],
-                        env.config["paths"]["worktrees"],
-                        "gecko"
-                        "autoland")
-    git_gecko.git.worktree("add",
-                           path,
-                           env.config["gecko"]["refs"]["autoland"])
+    path = os.path.join(env.config["root"], env.config["paths"]["worktrees"], "geckoautoland")
+    git_gecko.git.worktree("add", path, env.config["gecko"]["refs"]["autoland"])
     return git.Repo(path)
 
 
 @pytest.fixture
 def wpt_worktree(env, git_wpt):
     def inner(branch="test"):
-        path = os.path.join(env.config["root"],
-                            env.config["paths"]["worktrees"],
-                            "web-platform-tests"
-                            "test")
-        git_wpt.git.worktree("add",
-                             path,
-                             "origin/master")
+        path = os.path.join(
+            env.config["root"], env.config["paths"]["worktrees"], "web-platform-teststest"
```

Comment:
Here we also probably miss the comma. I guess it's probably not so important since it's the test data, but I guess we would still like to have this data close to the production